lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
This one is indeed a bit tricky.

I'll study what it's supposed to handle and let you know what direction to go.